File: log/src/main/java/no/nav/familie/log/NavHttpHeaders.java
package no.nav.familie.log;

public enum NavHttpHeaders {
    NAV_PERSONIDENT("Nav-Personident"),
    NAV_CALL_ID("Nav-Call-Id"),
    NAV_CONSUMER_ID("Nav-Consumer-Id");

    private final String header;

    NavHttpHeaders(String header) {
        this.header = header;
    }

    public String asString() {
        return header;
    }
}

File: log/src/main/java/no/nav/familie/log/NavHttpHeaders.kt
package no.nav.familie.log

enum class NavHttpHeaders(
    private val header: String,
) {
    NAV_PERSONIDENT("Nav-Personident"),
    NAV_CALL_ID("Nav-Call-Id"),
    NGNINX_REQUEST_ID("X-Request-Id"),
    NAV_CONSUMER_ID("Nav-Consumer-Id"),
    NAV_USER_ID("Nav-User-Id"),
    ;

    fun asString(): String = header
}
